charts/openstack-cloud-controller-manager/templates/daemonset.yaml

```diff
@@ -95,6 +95,9 @@ spec:
       initContainers: {{ toYaml .Values.extraInitContainers | nindent 6 }}
       {{- end }}
       hostNetwork: true
+      {{- if .Values.priorityClassName }}
```

Comment:
you need to increase the chart version in https://github.com/kubernetes/cloud-provider-openstack/blob/master/charts/openstack-cloud-controller-manager/Chart.yaml#L7 use like alpha.2